README.md

```diff
@@ -87,58 +87,19 @@ Maintainers ([@open-telemetry/collector-contrib-maintainer](https://github.com/o
-| honeycombexporter              | @paulosman @lizthegrey     |
-| jaegerthrifthttpexporter       | @jpkrohling @pavolloffay   |
-| kinesisexporter                | @owais                     |
-| newrelicexporter               | @MrAlias                   |
-| sapmexporter                   | @owais @dmitryax           |
-| sentryexporter                 | @AbhiPrasad                |
-| signalfxexporter               | @pmcollins @asuresh4       |
-| splunkhecexporter              | @atoulme                   |
-| stackdriverexporter            | @nilebox @james-bebbington |
-
-### Receivers
-
-| Receiver                       | Reviewer(s)               |
-| ------------------------------ | ------------------------- |
-| awsecscontainermetricsreceiver | @kbrockhoff @anuraaga     |
-| awsxrayreceiver                | @kbrockhoff @anuraaga     |
-| carbonreceiver                 | @pjanotti                 |
-| collectdreceiver               | @owais                    |
-| dockerstatsreceiver            | @rmfitzpatrick            |
-| k8sclusterreceiver             | @asuresh4                 |
-| kubeletstatsreceiver           | @pmcollins @asuresh4      |
-| prometheusexecreceiver         | @keitwb @james-bebbington |
-| receivercreator                | @jrcamp                   |
-| redisreceiver                  | @pmcollins @jrcamp        |
-| sapmreceiver                   | @owais                    |
-| signalfxreceiver               | @pjanotti @asuresh4       |
-| simpleprometheusreceiver       | @asuresh4                 |
-| statsdreceiver                 | @keitwb @jmacd            |
-| wavefrontreceiver              | @pjanotti                 |
-
-### Processors
-
-| Processor                      | Reviewer(s)               |
-| ------------------------------ | ------------------------- |
-| groupbytrace                   | @jpkrohling               |
-| routing                        | @jpkrohling               |
-
-### Extensions
-
-| Extension                      | Reviewer(s)               |
-| ------------------------------ | ------------------------- |
-| httpforwarder                  | @asuresh4                 |
-| jmxmetrics                     | @rmfitzpatrick            |
-| observer                       | @asuresh4 @jrcamp         |
+## PRs and Reviews
+
+When creating a PR please following the process [described
+here](https://github.com/open-telemetry/opentelemetry-collector/blob/master/CONTRIBUTING.md#how-to-structure-prs-to-get-expedient-reviews).
+
+News PRs will be automatically associated with the reviewers based on
+[CODEOWNERS](.github/CODEOWNERS). PRs will be also automatically assigned to one of the
+maintainers or approvers for facilitation. The facilitator is not tasked with reviewing
+(unless they also happen to be the codeowner), they are responsible for helping the
+PR author and reviewers to make progress or if progress cannot be made for closing the
+PR.
```

Comment:
The facilitator I think needs to do a review as well after the component owner reviewed it, if the component owner is not one of the "approvers". Does that make sense? Otherwise one of us (9) will need to do an official review anyway

Reply:
Edited, PTAL.